src/catkin_pkg/package.py

```diff
@@ -137,6 +137,18 @@ def has_buildtool_depend_on_catkin(self):
         """
         return 'catkin' in [d.name for d in self.buildtool_depends]
 
+    def build_type(self):
+        """
+        Returns value of export/build_type element, or 'catkin' if unspecified.
+
+        :returns: package build type
+        :rtype: str
+        """
+        for e in self.exports:
+            if e.tagname == 'build_type':
+                return e.content
+        return 'catkin'
```

Comment:
I think this is fine. If you're worried about it getting confused with the properties you could just call it `get_build_type()`. I'm fine with either.